### ISSUE TITLE
Run error on my own dataset

### ISSUE THREAD

**Issue by sonnefred** [kcheng1021/GaussianPro#54]
Hi, when I run the model on my own dataset, the following error occurs, and this is a random error during the training process. I checked the resolution of training images and they are the same. Could you give any help? Thanks.
![图片](https://github.com/kcheng1021/GaussianPro/assets/66628395/d0980df9-229d-4bc8-bb0a-c7b000f584cf)

**Comment by kcheng1021** [comment]
Hi, could you print the shape of normal?

**Comment by sonnefred** [comment]
> Hi, could you print the shape of normal?

Hi, I have printed the shape of normals, but sometimes the error occurs in another place, like this.
![图片](https://github.com/kcheng1021/GaussianPro/assets/66628395/19019563-fbe6-4c4d-8594-d922de287bf6)

**Comment by sonnefred** [comment]
I also encountered this error. It seems there is something wrong with tensor shapes ... Do you have any idea what may cause this? Thanks.
![图片](https://github.com/kcheng1021/GaussianPro/assets/66628395/f185cc93-4839-40aa-b09d-613ffc7dfe1f)

**Comment by sonnefred** [comment]
I print tensor shapes here:
![图片](https://github.com/kcheng1021/GaussianPro/assets/66628395/bf0367f6-06b1-4d01-a736-696e49c2631a)
And got error like this, and the shape of depth_src suddenly changes. Do you know what the reason is? 
![图片](https://github.com/kcheng1021/GaussianPro/assets/66628395/7b76ed25-8c43-4b46-9014-3920a58183d5)

**Comment by kcheng1021** [comment]
Sorry for the late reply. Do you ensure that your images in your dataset are in the same size?

**Comment by sonnefred** [comment]
> Sorry for the late reply. Do you ensure that your images in your dataset are in the same size?

Yes, I have checked that. The problem is that for the same scene data, the training can be finished after many attempts with the abovementioned error occurring, so maybe it's not related to the images resolution.

**Comment by kcheng1021** [comment]
I haven't encounter the error. Maybe you could upload your data for me to reproduce the error. 
Anyone else meet the same error is also welcome to discuss in this issue.

**Comment by sonnefred** [comment]
> I haven't encounter the error. Maybe you could upload your data for me to reproduce the error. Anyone else meet the same error is also welcome to discuss in this issue.

Hi, in previous issues I also see other people encountered the same error. I will do some experiments on Waymo and see whether the error exists. Thanks.

**Comment by JohnnyKimura** [comment]
hello friend，i wonder how can i run my own dataset？ i prepare my data use colmap but i got this error：

File "train.py", line 372, in <module> training(lp.extract(args), op.extract(args), pp.extract(args), args.test_iterations, args.save_iterations, args.checkpoint_iterations, args.start_checkpoint, args.debug_from) File "train.py", line 41, in training scene = Scene(dataset, gaussians) File "D:\01ZPS_data\models\GaussianPro\scene\__init__.py", line 74, in __init__ self.train_cameras[resolution_scale] = cameraList_from_camInfos(scene_info.train_cameras, resolution_scale, args) File "D:\01ZPS_data\models\GaussianPro\utils\camera_utils.py", line 77, in cameraList_from_camInfos camera_list.append(loadCam(args, id, c, resolution_scale)) File "D:\01ZPS_data\models\GaussianPro\utils\camera_utils.py", line 71, in loadCam sky_mask=resized_sky_mask, normal=resized_normal, depth=resized_depth) File "D:\01ZPS_data\models\GaussianPro\scene\cameras.py", line 53, in __init__ [0, K[1], K[3]], IndexError: index 3 is out of bounds for axis 0 with size 3

if you dont mind ，could you tell me how u prepare the dataset？ especially how you set up the dataset floder？  Thank you！ @sonnefred
